### DISCUSSION TITLE
Infinite News Feed

### DISCUSSION THREAD

**Discussion by JaySingh23** [reactplay/react-play#917] [Ideas]
Can anyone help me on how to build this? I am interested in builiding this project.

**Comment by atapas** [comment]
Hey @JaySingh23 can you please join our discord so that we can respond to you more real time?

**Reply by JaySingh23** [reply]
Yaa sure

**Comment by arcteggzz** [comment]
Please what is the link to the discord? I would like to join also.

**Comment by arcteggzz** [comment]
@atapas I also want to contribute to this particular Idea. But I need a few pointers. Is there any guide I can read?
